worlds/sm64ex/Regions.py

```diff
@@ -167,16 +167,10 @@ def create_regions(world: MultiWorld, player: int):
     world.regions.append(regBitS)
 
 
-def connect_regions(world: MultiWorld, player: int, source: str, target: str, rule=None):
+def connect_regions(world: MultiWorld, player: int, source: str, target: str, rule=None, name: str = None):
```

Comment:
iirc `connect_regions` is now available in one of the base classes right? If you're already changing it, could you just delete the function definition altogether and replace it with the new one?

Reply:
They're using one of the new connect helpers, which is as close to the function you call there as we have. Moving them to use one of the new helpers directly would require a full rewrite of the file.

Reply:
Nvm, I misread the change. What is done now looks exactly like I wanted.
Thats what I get for trying to read the diff from my phone ^^'